hazelcast/src/main/java/com/hazelcast/internal/bootstrap/TpcServerBootstrap.java

```diff
@@ -284,7 +279,6 @@ public void shutdown() {
 
         logger.info("TcpBootstrap shutdown");
 
-        shutdown = true;
```

Comment:
What happened this this field?

Reply:
It was unused. We can add shutdown field when we are going to implement functionality. WDYT?

Reply:
Remove what is unused.